release_notes.md

```diff
@@ -1,11 +1,23 @@
 # What's New
 
-- This solution pack is now **Uncertified**
+<table>
+    <tr>
+        <td><strong>NOTE</strong></td>
+        <td>This solution pack is now uncertified.</td>
```

Comment:
@cs-abhishek-shukla Please remove this. The solution pack is now going certified

Reply:
Fixed @rohit